docs/content/latest/admin/yb-tserver.md

```diff
@@ -25,53 +25,283 @@ $ ./bin/yb-tserver \
-`--bytes_durable_wal_write_mb` | N | 1MB | When `durable_wal_write` is false, writes to the Raft log are synced to disk every `bytes_durable_wal_write_mb` MB or `interval_durable_wal_write_ms` milliseconds whichever comes first.
+## Syntax
+
+```sh
+yb-tserver [ options ]
+```
+
+## Configuration options
+
+### General options
+
+#### --help
+
+Displays help on all options.
+
+#### --helpon
+
+Displays help on modules named by the specified option (or flag) value.
+
+#### --flagfile
+
+Specifies the file to load options (or flags) from.
+
+#### --version
+
+Shows version and build info, then exits.
+
+#### --tserver_master_addrs
+
+Comma-separated list of all the `yb-master` RPC addresses. Mandatory.
+
+#### --fs_data_dirs
+
+Comma-separated list of directories where the `yb-tserver` will place it's `yb-data/tserver` data directory. Mandatory.
+
+#### --fs_wal_dirs
+
+Default: Same as `--fs_data_dirs`
+
+The directory where the `yb-tserver` will place its write-ahead logs. May be the same as one of the directories listed in `--fs_data_dirs`, but not a sub-directory of a data directory.
+
+#### --max_clock_skew_usec
+
+Specifies the expected maximum clock skew between any two nodes in your deployment.
+
+Default: `50000` (50ms)
+
+#### --rpc_bind_addresses
+
+Specifies the comma-separated list of addresses to bind to for RPC connections.
+
+Default: `0.0.0.0:9100`
+
+#### --server_broadcast_addresses
+
+Public IP or DNS hostname of the server (along with an optional port).
+
+Default: `0.0.0.0:9100`
+
+#### --use_private_ip
+
+Determines when to use private IP addresses. Possible values are `never` (default),`zone`,`cloud` and `region`. Based on the values of the [placement (`--placement_*`) configuration options](#placement-options).
+
+Default: `never`
+
+#### --webserver_interface
+
+Address to bind for the web server user interface.
+
+Default: `0.0.0.0`
+
+#### --webserver_port
+
+The port for monitoring the web server.
+
+Default: `7000`
+
+#### --webserver_doc_root
+
+Monitoring web server home.
+
+Default: The `www` directory in the YugabyteDB home directory.
+
+### Logging options
+
+#### --logtostderr
+
+Flag to log to standard error (`stderr`).
+
+#### --log_dir
+
+Specifies the directory to store `yb-tserver` log files.
+
+Default: Same as [`--fs_data_dirs`](#fs-data-dirs)
+
+### Placement options
+
+#### --placement_zone
+
+Specifies the name of the availability zone, or rack, where this instance is deployed.
+
+Default: `rack1`
+
+#### --placement_region
+
+Specifies the name of the region, or data center, where this instance is deployed.
+
+Default: `datacenter1`
+
+#### --placement_cloud
+
+Specifies the name of the cloud where this instance is deployed.
+
+Default: `cloud1`
+
+### YSQL options
+
+The following options, or flags, support the use of the [YSQL API](../../api/ysql/).
+
+#### --enable_ysql
```

Comment:
Just a note: This will default to `true` soon in the 2.0.2 release.

Reply:
Got it -- can't wait :-)